src/main/java/com/bookstore/controller/OrderController.java

```diff
@@ -0,0 +1,68 @@
+package com.bookstore.controller;
+
+import com.bookstore.dto.order.CreateOrderRequestDto;
+import com.bookstore.dto.order.OrderResponseDto;
+import com.bookstore.dto.order.UpdateStatusRequestDto;
+import com.bookstore.dto.orderitem.OrderItemResponseDto;
+import com.bookstore.service.order.OrderService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Pageable;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "User's orders management", description = "Endpoints for managing user's orders")
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/orders")
+public class OrderController {
+    private final OrderService orderService;
+
+    @GetMapping
+    @Operation(summary = "Get all orders",
+            description = "Retrieve user's order history")
+    public List<OrderResponseDto> getAll(Pageable pageable) {
+        return orderService.findAll(pageable);
```

Comment:
Use get... naming for all methods on this service and controller levels. Change in others methods.